acados/ocp_nlp/ocp_nlp_common.h

```diff
@@ -93,69 +85,84 @@ typedef struct
 	struct blasfeo_dmat *W;
     struct blasfeo_dvec *y_ref;
 	int *nls_mask; // nonlinear least squares mask
-	int memsize;
 } ocp_nlp_cost_ls;
```

Comment:
This is preferably defined in the specific ``ocp_lin`` linearization module (GGN) instead of ``ocp_nlp_common.h``. Since the separation between NLP methods and linearization methods are not yet made: ``ocp_nlp_sqp_gn.h`` is most appropriate place atm. 

**Reason:** we do not yet know what kind of exotic costs we (or anyone else) wants to support in the future. This is only a concern to the specific linearization module(s) that decide(s) to support it (such as GGN supporting least-squares stage costs, and EH supporting general stage costs).